### ISSUE TITLE
Option to show horizontal scroll bar

### ISSUE THREAD

**Issue by cuijulian** [visjs/vis-timeline#1002]
When using `verticalScroll: true` it shows a vertical scroll bar at the left side of the timeline.

I would like to request a feature to show a horizontal scroll bar on the top or bottom of the timeline to scroll through times:

![horizontal_scroll_bar](https://user-images.githubusercontent.com/28796012/113180481-c7946f00-921e-11eb-9cf2-a172a81a1c8e.png)

Is this already supported? Thanks in advance!

**Comment by mojoaxel** [comment]
I don't think that a horizontal scrollbar makes sence here because for this to make sence the `min` and `max` would have to be set. Otherwise the timeline can be scrolled endlessly.

I would recommend hinting the user somehow to use the mouse-drag feature.

**Comment by CrashLaker** [comment]
we could maybe scroll horizontally when "Shift" is pressed

no key scroll: scroll vertically
shift + scroll: scroll horizontally
ctrl + scroll: zoom

**Comment by ChR-iSz** [comment]
Any news here ? A horizontally scrollbar sounds amazing....

**Comment by simonbpro** [comment]
Hello, here is a hack to have a horizontal scrollbar with a div surrounding the actual timeline. 
Need to use min and max to have the component set with boundaries ! 
https://codepen.io/simonflutilliant/pen/RwxoZRY

**Comment by cnagytcorp** [comment]
"shift + scroll: scroll horizontally" I like the idea, recently turned on fixed height now lost horizontal scroll

**Comment by BDMChau** [comment]
have any update, I also need this feature

**Comment by mojoaxel** [comment]
A horizontal scrollbars only would make sense if min and max is defined.

It could not be implemented as a normal scroll bar but basically just as an indicator of the current scroll position and zoom level.

This library is basically unmaintained at the moment.

Don't expect any new features like this without external contribution.

If your company needs this: I would estimate ~4k€ to implement this.

**Comment by ChR-iSz** [comment]
@mojoaxel

Are you serious to ask for money in threads to implement a scrollbar. I will draw my conclusions and not follow this repository anymore. 4k for a scrollbar! That's a joke...

**Comment by asinghca** [comment]
Firstly, I appreciate the authors that put in their valuable time to give us this comprehensive timeline library to use for free of cost.

Here is a great example that one can leverage in building their scrollbar functionality via move left/right buttons: https://visjs.github.io/vis-timeline/examples/timeline/interaction/navigationMenu.html

Again, thank you to all the authors for putting in extra time and care in drafting all these detailed examples!

**Comment by Amraj-Aujla** [comment]
Any Update on this?
